internal/pkg/db/db_test.go

```diff
@@ -55,10 +56,10 @@ func (dbs *dbSuite) SetupSuite() {
 		dbs.documents = append(dbs.documents, doc)
 	}
 
-	dbs.backend = db.NewBackend()
-	dbs.backend.Options.DatabaseFile = db.DatabaseFile
+	dbs.backend, err = db.NewBackend(&options.Options{})
 
-	if err := dbs.backend.InitClient(); err != nil {
+	dbs.backend.Options.DatabaseFile = db.DatabaseFile
+	if err != nil {
 		dbs.T().Fatalf("%v", err)
 	}
```

Comment:
It's not letting me use a suggestion, but I recommend

```go
	dbs.backend, err = db.NewBackend(&options.Options{})
	if err != nil {
		dbs.T().Fatalf("%v", err)
	}

	dbs.backend.WithDatabaseFile(db.DatabaseFile)
```

or if trying out the suggested changes above:

```go
	dbs.backend, err = db.NewBackend(db.WithDatabaseFile(db.DatabaseFile))
	if err != nil {
		dbs.T().Fatalf("%v", err)
	}
```